docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -1087,6 +1087,10 @@ Task queue configuration has been moved to `queue.task`. However, the below conf
 
 - `ENABLED`: **true**: Enable/Disable federation capabilities
 - `SHARE_USER_STATISTICS`: **true**: Enable/Disable user statistics for nodeinfo if federation is enabled
+- `ALGORITHMS`: **rsa-sha256, rsa-sha512**: HTTP signature algorithms
+- `DIGEST_ALGORITHM`: **SHA-256**: HTTP signature digest algorithm
+- `GET_HEADERS`: **(request-target), Date**: GET headers for federation requests
```

Comment:
isn't this an implementation detail?

Reply:
I guess so, because if an instance admin changes these settings, it will break federation with other AP software like Mastodon. (However, that instance will still be able to federate with other Gitea instances. This is because we use go-fed/httpsig for HTTP signatures which supports many algorithms, but Mastodon, for instance, only supports `rsa-sha256` and `hs2019` which it handles the exact same way as `rsa-sha256`.)

Reply:
I was revering to GET_HEADERS and POST_HEADERS here

Reply:
Yeah, I can't think of any use cases where an instance admin would want to change the `GET_HEADERS` and `POST_HEADERS` settings.